src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+a:visited {
+  color: inherit;
+  text-decoration: none;
+}
+
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: rgba(0, 172, 220, 1);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  transition: 300ms;
+
+  &:hover {
+    background-color: rgba(255, 255, 255, 1);
+    color: rgba(0, 172, 220, 1);
+    border: 1px solid rgba(0, 172, 220, 1);
+  }
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  transition: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    padding-inline: 3px;
+    min-height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: rgba(97, 96, 112, 1);
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 14px;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+  }
+}
```

Comment:
The `.card` class has a hover effect that scales the card. Ensure this effect does not cause layout shifts that could affect user experience.